agenta-backend/agenta_backend/migrations/mongo_to_postgres/migration.py

```diff
@@ -0,0 +1,522 @@
+
+tables = [
+    UserDB,
+    ImageDB,
+    AppDB,
+    DeploymentDB,
+    VariantBaseDB,
+    AppVariantDB,
+    AppVariantRevisionsDB,
+    AppEnvironmentDB,
+    AppEnvironmentRevisionDB,
+    TemplateDB,
+    TestSetDB,
+    EvaluatorConfigDB,
+    HumanEvaluationDB,
+    HumanEvaluationScenarioDB,
+    EvaluationDB,
+    EvaluationScenarioDB,
+    IDsMappingDB,
+    EvaluationEvaluatorConfigDB,
+    EvaluationScenarioResultDB,
+]
+
+
+async def transform_user(user):
+    user_uuid = generate_uuid()
+    await store_mapping("users", user["_id"], user_uuid)
+    return {
+        "id": user_uuid,
+        "uid": user["uid"],
+        "username": user["username"],
+        "email": user["email"],
+        "created_at": get_datetime(user.get("created_at")),
+        "updated_at": get_datetime(user.get("updated_at")),
+    }
+
+
+async def transform_image(image):
+    user_uuid = await get_mapped_uuid(
+        image["user"].id if isinstance(image["user"], DBRef) else image["user"]
+    )
+    image_uuid = generate_uuid()
+    await store_mapping("docker_images", image["_id"], image_uuid)
+    return {
+        "id": image_uuid,
+        "type": image["type"],
+        "template_uri": image.get("template_uri"),
+        "docker_id": image.get("docker_id"),
+        "tags": image.get("tags"),
+        "deletable": image.get("deletable", True),
+        "user_id": user_uuid,
+        "created_at": get_datetime(image.get("created_at")),
+        "updated_at": get_datetime(image.get("updated_at")),
+    }
+
+
+async def transform_app(app):
+    user_uuid = await get_mapped_uuid(app["user"].id)
+    app_uuid = generate_uuid()
+    await store_mapping("app_db", app["_id"], app_uuid)
+    return {
+        "id": app_uuid,
+        "app_name": app["app_name"],
+        "user_id": user_uuid,
+        "created_at": get_datetime(app.get("created_at")),
+        "updated_at": get_datetime(app.get("updated_at")),
+    }
+
+
+async def transform_deployment(deployment):
+    app_uuid = await get_mapped_uuid(deployment["app"].id)
+    user_uuid = await get_mapped_uuid(deployment["user"].id)
+    deployment_uuid = generate_uuid()
+    await store_mapping("deployments", deployment["_id"], deployment_uuid)
+    return {
+        "id": deployment_uuid,
+        "app_id": app_uuid,
+        "user_id": user_uuid,
+        "container_name": deployment.get("container_name"),
+        "container_id": deployment.get("container_id"),
+        "uri": deployment.get("uri"),
+        "status": deployment["status"],
+        "created_at": get_datetime(deployment.get("created_at")),
+        "updated_at": get_datetime(deployment.get("updated_at")),
+    }
+
+
+async def transform_variant_base(base):
+    app_uuid = await get_mapped_uuid(base["app"].id)
+    user_uuid = await get_mapped_uuid(base["user"].id)
+    image_uuid = await get_mapped_uuid(base["image"].id)
+    deployment_uuid = base["deployment"] and await get_mapped_uuid(base["deployment"])
```

Comment:
I don't understand this line, it's confusing. It looks like deployment_uuid is a bool (since you are using and) but it is later handled as an id.